README.md

```diff
@@ -398,6 +398,48 @@ HTTP call to the platform:<br/>
   }
   ```
 
+## Interaction Log service
```

Comment:
Can this go in a separate markdown file somewhere or at the very least create a table of contents for the main doc? Our main readme is becoming quite large and without any quick links it just looks daunting rather than helpful

Reply:
Moved interaction logs readme to a new md file, added that new file inside DOCS